test/system/src/test/java/io/pravega/test/system/WatermarkingTest.java

```diff
@@ -153,13 +151,6 @@ public void watermarkingTests() throws Exception {
         Stream streamObj = Stream.of(SCOPE, STREAM);
         scale(controller, streamObj);
 
-        // wait until mark stream is created
```

Comment:
You remove statements in lines 157-161. What is the impact on the test? Given the lines below, it looks like the test is excepted to function without these lines now. But then the question is why were they needed in the first place. 

Am I missing something?

Reply:
@shiveshr Would be able to explain better, but he mentioned two points to me about this test:

1. The `WatermarkingStream` was originally created lazily so the test had to wait for it to be created, but this is not the case anymore.
2. A store exception is never thrown back to the client, so the exception being compared to has no meaning.